### DISCUSSION TITLE
Host kernel and guest kernel to try firecracker

### DISCUSSION THREAD

**Discussion by ashwin31** [firecracker-microvm/firecracker#4886] [Q&A]
Do I need host kernel 6.1 to try guest vm based on kernel 6.1?

If yes, how to get host kernel specific kernel 6.1. I tried on digitalocean and getting 6.8 when I do upgrade.

There is no os I can find with exact 6.1 kernel.

**Comment by JackThomson2** [comment]
Hi @ashwin31,

Our official kernel support policy can be found [here](https://github.com/firecracker-microvm/firecracker/blob/main/docs/kernel-policy.md). However, other kernel versions, although not validated, might work. So you are able to try running with a 6.1 guest on the 6.8 host.
